projects/lf-documentation/src/app/getting-started-npm-example/getting-started-npm-example.component.html

```diff
@@ -0,0 +1,48 @@
+<h3 id="npm-example">Example: <code>lf-checklist</code> In an Angular Project Via NPM
+</h3>
+<p>In the example below, both the implementation and the typing are provided via <code>@laserfiche/lf-ui-components/lf-checklist</code>.</p>
+<ol>
+  <li>
+    <p><code>npm install @laserfiche/lf-ui-components</code></p>
+  </li>
+  <li>
+    <p>
+      Inside <code>./app.module.ts</code>, import the Angular modules.
+    </p>
+      <pre  class="code-container">
+<span class="code-ts-import">import</span> &#123; NgModule } <span class="code-ts-import">from</span> <span class="code-ts-import-source">'@angular/core'</span>;
+<span class="code-ts-import">import</span> &#123; LfChecklistModule } <span class="code-ts-import">from</span> <span class="code-ts-import-source">'@laserfiche/lf-ui-components/lf-checklist'</span>;
+@NgModule(&#123;
+  <span class="code-ts-key-name">declarations:</span> [
+    AppComponent,
+    ...
+  ],
+  <span class="code-ts-key-name">imports: </span>[
+    LfChecklistModule,
+    BrowserAnimationsModule
+  ]
+})
+      </pre>
+
+    </li>
+    <li>
+      <p dir="auto">
+        Import the style sheet from the npm package by adding this line in <code>styles.css</code> in your
+        angular application.
+      </p>
+
+      <pre class="code-container">
+<span class="code-ts-import">@import</span> '~@laserfiche/lf-ui-components/cdn/lf-laserfiche-lite.css';
+      </pre>
+    </li>
+      <li>
+        The same usage of using <code>lf-checklist</code> in the html and ts file as used as CDN, except to import the component typing from <code>@laserfiche/lf-ui-components</code>
+
+        <pre class="code-container">
+<span class="code-ts-import">import</span> &#123; AfterViewInit, Component, ViewChild, ElementRef } <span class="code-ts-import">from</span> <span class="code-ts-import-source">'@angular/core'</span>;
```

Comment:
I would copy the implementation here since it's on a separate page... or link to the other page's section